lib/modules/manager/gradle/parser.ts

```diff
@@ -895,7 +919,7 @@ export function parseGradle(
 
 const propWord = '[a-zA-Z_][a-zA-Z0-9_]*(?:\\.[a-zA-Z_][a-zA-Z0-9_]*)*';
 const propRegex = regEx(
-  `^(?<leftPart>\\s*(?<key>${propWord})\\s*=\\s*['"]?)(?<value>[^\\s'"]+)['"]?\\s*$`
+  `^(?<leftPart>\\s*(?<key>${propWord})\\s*[= :]\\s*['"]?)(?<value>[^\\s'"]+)['"]?\\s*$`
```

Comment:
correct me if i'm wrong, i have a small doubt,
i believe `[=:]` should be enough here why is there a space in between? we already have `\\s*` before and after,
and if you mean the `=` and `:` are Optional, you can add `[=:]?`

Reply:
Had the same thought initially but using `[=:]?` could have unwanted some side effects:

- `\s*` matches {0, *} times. With `\\s*[=:]?\\s*` a space may be omitted overall. With no separator between key and value matching fails
- Using `\s+` left or right to `[=:]?` would be a remedy for this specific case but that, in turn, would prevent matching legit statements without space, like `key=value`

Hence, making the space explicit in `[= :]` seemed like the best option for me.

Reply:
Didn't think of that, maybe we should add a comment in code so no one would change it to that then

Reply:
I'm not really worried this may be happen because there are tests covering `key=value` and `key value` 👍

Reply:
oh Lovely!